modules/cookbooks/pages/kafka_migrator.adoc

```diff
@@ -0,0 +1,303 @@
+= Kafka Migrator
+
+// tag::single-source[]
+
+Author: https://www.linkedin.com/in/mtodor[Mihai Todor^]
+
+:description: Kafka Migrator is a tool for simplifying migrations from any Kafka system to Redpanda. Its purpose is to allow users to migrate Kafka messages, schemas and ACLs in a manner that is both user-friendly and efficient, while also offering a high degree of customizability.
+
+The Redpanda Connect components which enable this functionality are the xref:components:inputs/kafka_migrator.adoc[`kafka_migrator` input], xref:components:outputs/kafka_migrator.adoc[`kafka_migrator` output], xref:components:inputs/kafka_franz.adoc[`kafka_franz` input], xref:components:outputs/kafka_migrator_offsets.adoc[`kafka_migrator_offsets` output], xref:components:inputs/schema_registry.adoc[`schema_registry` input] and xref:components:outputs/schema_registry.adoc[`schema_registry` output]. For convenience, these are bundled together in the xref:components:inputs/kafka_migrator_bundle.adoc[`kafka_migrator_bundle` input] and xref:components:outputs/kafka_migrator_bundle.adoc[`kafka_migrator_bundle` output] templates which we will be demoing in this cookbook.
+
+== Setup
+
+First, we will need two clusters and, for simplicity, we can run two Redpanda Docker containers via https://docs.docker.com/compose[Docker Compose^]:
+
+[source,yaml]
+----
+# docker-compose.yml
+
+services:
+  source:
+    image: redpandadata/redpanda
+    command:
+      - redpanda
+      - start
+      - --node-id 0
+      - --mode dev-container
+      - --set rpk.additional_start_flags=[--reactor-backend=epoll]
+      - --set redpanda.auto_create_topics_enabled=false
+      - --kafka-addr 0.0.0.0:9092
+      - --advertise-kafka-addr localhost:9092
+      - --schema-registry-addr 0.0.0.0:8081
+    ports:
+      - 8081:8081
+      - 9092:9092
+      - 9644:9644
+
+  sink:
+    image: redpandadata/redpanda
+    command:
+      - redpanda
+      - start
+      - --node-id 0
+      - --mode dev-container
+      - --set rpk.additional_start_flags=[--reactor-backend=epoll]
+      - --set redpanda.auto_create_topics_enabled=false
+      - --kafka-addr 0.0.0.0:9093
+      - --advertise-kafka-addr localhost:9093
+      - --schema-registry-addr 0.0.0.0:8081
+    ports:
+      - 8082:8081
+      - 9093:9093
+      - 9645:9644
+----
+
+[source,console]
+----
+$ docker-compose -f docker-compose.yml up --force-recreate -V
+----
+
+Once these are up and running, we can use the Redpanda https://docs.redpanda.com/current/get-started/rpk[`rpk` CLI tool^] to interact with them and create two topics and a schema for each of them in the `source` cluster. Also, we create a role and, for each topic, we add an ACL.
```

Comment:
```suggestion
Once the clusters are up and running, execute the following https://docs.redpanda.com/current/get-started/rpk[`rpk`] commands to interact with them and create two topics and a schema for each of them in the `source` cluster. These commands also create a security role and an ACL for each topic.
```

Reply:
OK, I moved the `rpk` section further down and mentioned that it needs to be installed.